crates/biome_css_analyze/tests/specs/nursery/noUnknownPseudoClassSelector/valid.css

```diff
@@ -0,0 +1,36 @@
+a:hover { }
+a:Hover { }
+a:hOvEr { }
+a:HOVER { }
+a:focus-visible { }
+a:before { }
+a::before { }
+:modal { }
+input:not([type='submit']) { }
+:matches(section, article, aside, nav) h1 { }
+a:has(> img) { }
+section:has(h1, h2, h3, h4, h5, h6) { }
+:root { }
+p:has(img):not(:has(:not(img))) { }
+div.sidebar:has(*:nth-child(5)):not(:has(*:nth-child(6))) { }
+div :nth-child(2 of .widget) { }
+a:hover::before { }
+a:-moz-placeholder { }
+a,\nb > .foo:hover { }
```

Comment:
```suggestion
a,
b > .foo:hover { }
```

Reply:
I fixed it in https://github.com/biomejs/biome/commit/31441057cb0fca66508afb155054ca170fe32894 and update snapshots